content/bn/docs/concepts/architecture/garbage-collection.md

```diff
@@ -0,0 +1,136 @@
+---
+title: গার্বেজ কালেকশন
+content_type: concept
+weight: 70
+---
+
+<!-- overview -->
+{{<glossary_definition term_id="garbage-collection" length="short">}}
+গার্বেজ কালেকশন সিস্টেমকে নিম্নলিখিত রিসোর্সগুলি পরিষ্কার করতে দেয়:
+
+* [সমাপ্ত পডগুলি](/bn/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection)
+* [সম্পন্ন হওয়া জবগুলি](/bn/docs/concepts/workloads/controllers/ttlafterfinished/)
+* [মালিক রেফারেন্স ছাড়া অবজেক্ট](#owners-dependents)
+* [অব্যবহৃত কন্টেইনার এবং কন্টেইনার ইমেজগুলি](#containers-images)
+* [ডায়নামিকভাবে প্রস্তুতকৃত, StorageClass এর জন্য পুনরুদ্ধার নীতির সাথে Delete করা PV](/bn/docs/concepts/storage/persistent-volumes/#delete)
+* [ব্যর্থ অথবা মেয়াদ উত্তীর্ণ সার্টিফিকেট সাইনিং রিকোয়েস্ট (CSR)](/bn/docs/reference/access-authn-authz/certificate-signing-requests/#request-signing-process)
+* {{<glossary_tooltip text="নোড" term_id="node">}} নিম্নলিখিত পরিস্থিতিতে মুছে ফেলা :
+  * যখন ক্লাস্টারটি [ক্লাউড কন্ট্রোলার ম্যানেজার](/bn/docs/concepts/architecture/cloud-controller/) এর মাধ্যমে ক্লাউডে চালানো হয়।
+  * যখন ক্লাস্টারটি ক্লাউড কন্ট্রোলার ম্যানেজারের মতো প্লাগইন ব্যবহার করে লোকাল পরিবেশে চলে।
+* [নোড লিজ অবজেক্ট](/bn/docs/concepts/architecture/nodes/#heartbeats)
+
+## মালিক ও নির্ভরশীল {#owners-dependents}
+
+Kubernetes এ অনেক অবজেক্ট একে অপরের সাথে [**মালিকানাধীন রেফারেন্স**](/bn/docs/concepts/overview/working-with-objects/owners-dependents/) এর মাধ্যমে সংযুক্ত থাকে। মালিকানাধীন রেফারেন্স (Owner Reference) কন্ট্রোল প্লেনকে জানায় কোন অবজেক্ট অন্য অবজেক্টের উপর নির্ভরশীল। Kubernetes মালিকানাধীন রেফারেন্স ব্যবহার করে কন্ট্রোল প্লেন এবং অন্যান্য API ক্লায়েন্টদের একটি অবজেক্ট মুছে ফেলার সময় সংশ্লিষ্ট রিসোর্সগুলি পরিষ্কার করার একটি সুযোগ প্রদান করে। বেশিরভাগ পরিস্থিতিতে Kubernetes স্বয়ংক্রিয়ভাবে মালিকানাধীন রেফারেন্স পরিচালনা করে।
```

Comment:
Use the English version as reference `[**মালিকানাধীন রেফারেন্স**](/docs/concepts/overview/working-with-objects/owners-dependents/)`